src/arithmetic.rs

```diff
@@ -0,0 +1,110 @@
+use crate::conversion::Cast;
```

Comment:
add //! comments to the beginning of each rust file, clippy will also check this